BEST_PRACTICES.md

```diff
@@ -1,18 +1,32 @@
-# When to use blocks, delegates or data source
+# Best practices
 
-## Block
+## When to use blocks, delegates or data source
+
+### Block
 - Asynchronous (For example: networking operations)
 - User inputs with multiple options (For example: UIAlertView's YES and NO)
 - Data source driven inputs (For example: A table items with action blocks that were defined in the data source)
 - Returns many values (For example looking for a field in a collection and returning the field and the indexPath)
 - If there’s no tracked state or if state it’s defined in the same method
 
-## Delegate
+### Delegate
 - Synchronous (For example: buttons actions in views that should perform on their parents)
 - Shouldn't return values
 - Provides control over performing an action (For example: UITextField's shouldEndEditing)
 - User input with one action (For example: buttons actions in views that should perform on their parents)
 - If tracked state is shared (if state is stored in a property or a constant)
 
-## Data source
+### Data source
 - Returns ONE value
+
+## Deployment
+
+### Semantic Versioning
+
+We support [semantic versioning](http://semver.org/), and it's pretty important that minor releases are backwards compatible otherwise don't feel shy to make it a major version.
```

Comment:
Can we drop `pretty`, because I think it is important. If it wasn't then it wouldn't be a minor release.